config

```diff
@@ -183,6 +183,14 @@ USE_ADI_REPO_CARRIERS_BOOT=y
 #
 #
 #
+# [PREPARE BOOT FILES]*
+# Prepare boot partition with carriers boot files at build time by setting the PROJECT_NAME and the BOARD.
+# *depends on INTEL AND XILINX BOOT FILES
+PROJECT_NAME=
```

Comment:
I know that in kuiper.json, the nodes are 'name' and 'board', but I would change the variables names in config file to something more suggestive, like 'ADI_EVAL_BOARD' and 'CARRIER' or 'PLATFORM'.